src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -982,6 +982,51 @@ public static IEnumerable<object[]> GetAsync_IPBasedUri_Success_MemberData()
             });
         }
 
+        [Theory]
```

Comment:
@justinvp @stephentoub sorry, not sure about that. I do see that many tests in theHttpClientHandlerTest do have this attribute. Just let me know if [OuterLoop] should be there and I'll add it. THanks.